src/main/java/core/basesyntax/db/Storage.java

```diff
@@ -0,0 +1,29 @@
+package core.basesyntax.db;
+
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Set;
+
+public class Storage {
+    public static final Map<String, Integer> fruits = new HashMap<>();
+
+    public Integer getQuantity(String fruit) {

```

Comment:
The method 'getQuantity' might return null if the map does not contain the fruit. Consider returning an Optional<Integer> to avoid potential NullPointerExceptions.

Reply:
I added check condition in this method,